src/Packages/Passport/Runtime/ThirdParty/Gree/Assets/Plugins/WebViewObject.cs

```diff
@@ -64,7 +64,7 @@ public static Singleton Instance
     }
 #endif
 
-    public class WebViewObject
+    public class WebViewObject : MonoBehaviour
```

Comment:
Does it have to be a MonoBehaviour to work? On other platforms, does the object get destroyed when you change scenes? Can you please check it with the sample app, as we use scenes? The game is more like one scene that shows and hides views.

Reply:
In webgl, the webviewObject needs to be a gameObject so it can communicate with unityInstanceunityInstance.SendMessage(name, 'CallOnAuth', event.data.url);
I tested that it works well in other platforms, but let me update it to make it webgl sepecific anyway.